tutorials/eth-api/randomness-lottery.md

```diff
@@ -0,0 +1,500 @@
+---
+title: Create a Lottery with the Randomness Precompile
+description: Looking to create a lottery smart contract? Follow this step-by-step tutorial on using Moonbeam's Randomness Precompile (a Solidity interface) to get started.
+---
+
+# Create a Lottery Contract using the Randomness Precompile
+
+![Randomness Moonbeam Banner](/images/builders/pallets-precompiles/precompiles/randomness/randomness-banner.png)
+
+_March 15, 2022 | by Erin Shaben_
+
+## Introduction {: #introduction } 
```

Comment:
My main an only comment here is that probably adding a flow diagram of how the thing works may be helpful. I feel there is a LOT of text to go through